types/egg/index.d.ts

```diff
@@ -535,6 +535,20 @@ export interface Application extends EggApplication {
 
 interface FileStream extends Readable { // tslint:disable-line
   fields: any;
+  
```

Comment:
@Rwing The egg declaration file is bundling with npm package now. You can just send pr to the [egg](https://github.com/eggjs/egg) repo.